### PR TITLE
v4.4.1 - .u-showAboveNarrow class to include 414px in media query

### PR DESCRIPTION
**Pull request by dandel10n**
### Changed
- `.u-showAboveNarrow` class to include 414px in media query

### CODE REVIEW

**Review by DamianMullins — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 
+v4.4.1
+------------------------------
+*Fabruary 11, 2021*
```
Comment (kevinrodrigues) [inline CHANGELOG.md:9]: There's a typo here.

**Review by DamianMullins — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (dandel10n) [reply CHANGELOG.md]: Oh no... fixed :heavy_check_mark:

**Review by charliearlie — APPROVED** [review]

File: src/scss/trumps/_rwd.scss [hunk 2]
```diff
@@ -31,7 +31,7 @@
 }
 
 .u-showAboveNarrow {
-    @include media('<narrow') {
+    @include media('<=narrow') {
```
[the remark this answers is not in the document]
Comment (DamianMullins) [reply src/scss/trumps/_rwd.scss:34]: There does seem to be some inconsistency between these styles..

[the remark this answers is not in the document]
Comment (dandel10n) [reply src/scss/trumps/_rwd.scss:34]: Yes, true, I will test them and push the change for these to be consistent. For now I changed only the one which was causing problems on homepage

**Pull request merged**